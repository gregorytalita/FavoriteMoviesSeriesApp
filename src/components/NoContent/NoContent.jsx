import React from 'react'
import { Typography, makeStyles } from '@material-ui/core'
import { ReactComponent as None } from '../../assets/none.svg'

const useStyles = makeStyles(theme => ({
  wrapper: {
    display: 'flex',
    flexDirection: 'column',
    justifyContent: 'center',
    maxWidth: 600
  }
}))

const NoContent = () => {
  const classes = useStyles()

  return (
    <div className={classes.wrapper}>
      <Typography variant='h2' align='center'>
        You didnâ€™t add anything yet
      </Typography>
      <None />
    </div>
  )
}

export default NoContent
