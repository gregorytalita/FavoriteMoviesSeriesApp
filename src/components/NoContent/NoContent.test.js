import React from 'react'
import { create } from 'react-test-renderer'
import NoContent from './NoContent'

describe('NoContent component', () => {
  test('it matches the snapshot', () => {
    const component = create(<NoContent />)
    expect(component.toJSON()).toMatchSnapshot()
  })
})
