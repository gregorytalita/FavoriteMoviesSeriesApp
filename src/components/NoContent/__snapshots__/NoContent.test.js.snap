// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContent component it matches the snapshot 1`] = `
<div
  className="makeStyles-wrapper-1"
>
  <h2
    className="MuiTypography-root MuiTypography-h2 MuiTypography-alignCenter"
  >
    You didnâ€™t add anything yet
  </h2>
  <svg>
    none.svg
  </svg>
</div>
`;
